### PR TITLE
Added new field ClientPasswordAuthType

### PR DESCRIPTION
**Pull request by paramoshaws**
Added ClientPasswordAuthType field

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-rds-dbproxyendpoint/pom.xml [hunk 1]
```diff
@@ -61,11 +61,11 @@
             <version>2.26.0</version>
             <scope>test</scope>
         </dependency>
-
+        <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-rds -->
```
Comment (hex-y) [inline aws-rds-dbproxyendpoint/pom.xml:64]: why do we need to add this?

File: aws-rds-dbproxy/pom.xml [hunk 2]
```diff
@@ -68,10 +68,11 @@
             <version>2.26.0</version>
             <scope>test</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-rds -->
```
Comment (hex-y) [inline aws-rds-dbproxy/pom.xml:71]: why do we need to add this?

**Review by hex-y — APPROVED** [review]
have two small comments, also make sure to squash when merging the changes.

File: aws-rds-dbproxyendpoint/pom.xml (hunk 1 above)
Comment (paramoshaws) [reply aws-rds-dbproxyendpoint/pom.xml:64]: it's just a hint where to look up for versions available. We had this added for proxy targets and missed for proxies and endpoints.

**Review by zhuyibo-amzn — APPROVED** [review]

**Pull request merged**